whitesands/code/modules/clothing/head/jobs.dm

```diff
@@ -46,3 +46,23 @@
 	icon_state = "cowboyhos"
 
 	dog_fashion = /datum/dog_fashion/head/cowboy
+
+/obj/item/clothing/head/cowboy/sec/roumain
+	name = "hunter's hat"
+	desc = "A fancy hat with a nice feather. It covers your eyes, which makes you feel badass."
+	icon_state = "rouma_hat"
+
+/obj/item/clothing/head/cowboy/sec/roumain/shadow
+	name = "shadow's hat"
+	desc = "A okay looking hat. It covers your eyes, which makes you feel badass, but in reality you look like a wannabe hunter."
```

Comment:
desc = "A rough, simple hat. The way it covers your eyes makes you feel badass, but you just look like a wannabe hunter."